test/uk/gov/hmrc/mtdsatestsupportapi/connectors/DeleteCheckpointConnectorSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mtdsatestsupportapi.connectors
+
+import api.connectors.ConnectorSpec
+import api.models.domain.CheckpointId
+import api.models.errors.{DownstreamErrorCode, DownstreamErrors}
+import api.models.outcomes.ResponseWrapper
+import play.api.http.Status._
+import play.api.libs.json.Json
+import uk.gov.hmrc.mtdsatestsupportapi.models.request.deleteCheckpoint.DeleteCheckpointRequest
+
+class DeleteCheckpointConnectorSpec extends ConnectorSpec {
+
+  private val vendorClientId = "someVendorId"
+  private val checkpointId   = "someCheckpointId"
+  private val downstreamUri  = s"/test-support/vendor-state/$vendorClientId/checkpoints/$checkpointId"
+  private val requestData    = DeleteCheckpointRequest(vendorClientId, CheckpointId("someCheckpointId"))
```

Comment:
you could move these val to inside the Test trait. Make them protected or public so they can be accessed in the tests but not outside the file